src/components/AdminRegisterPage/index.jsx

```diff
@@ -40,7 +41,16 @@ const AdminRegisterPage = () => {
         logError(error);
       }
     };
-    getEnterpriseBySlug();
+    LmsApiService.loginRefresh().then(_ => {
+      const obfuscatedId = uuidv5(String(_.userId), uuidv5.DNS);
```

Comment:
Can you explain/comment why this needs to be obfuscated? Like, why not just put the user ID in the storageKey?